hipcub/PKGBUILD

```diff
@@ -2,46 +2,29 @@
 pkgname=hipcub
 pkgver=3.3.0
 pkgrel=2
-pkgdesc="Reusable software components for the rocm developers."
+pkgdesc='header-only library on top of rocPRIM or CUB'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/hipCUB"
-license=('custom:NCSAOSL')
-depends=('hcc' 'hip')
-makedepends=('cmake' 'python2' 'rocminfo' 'comgr')
-source=("https://github.com/ROCmSoftwarePlatform/hipCUB/archive/rocm-$pkgver.tar.gz")
+license=('custom')
+depends=('rocprim')
+optdepends=('cuda: CUB backend')
+makedepends=('cmake' 'git' 'hcc')
```

Comment:
Is cuda a compile option if so would we need to create a hipcub-rocm and a hipcub-cuda package?

Reply:
It seems so. But I don't have an nVidia GPU to test it.

Reply:
hmm kk this is fine for now then